dev/mapSTRING.R

```diff
@@ -0,0 +1,151 @@
+# Script to load STRING coexpression data and map it
```

Comment:
This is a part of the documentation. Two options: move this to ./inst/scripts after merge, or cover it in a vignette. ./dev/ is only for actual development.